### ISSUE TITLE
Waiting for data indication on main inputs

### ISSUE THREAD

**Issue by NirBenita** [flydelabs/flyde#108]
I noticed when _nodes inside a flow are waiting for data_, they have a purple indication. But when the _main inputs are waiting for data_, there is no indication. This makes it hard to understand what's going on in the flow...

I think there should be a similar indication on the main inputs as well 🙏

**Comment by GabiGrin** [comment]
@NirBenita good catch! adding an indication to main inputs should be trivial. Hope to get to it soon. Thanks!
